src/AppBundle/Resources/views/Ndr/Asset/_list.html.twig

```diff
@@ -2,31 +2,31 @@
 {% trans_default_domain translationDomain %}
 
 {# "client own assets" question #}
-<table class="check-your-answers columns-2 push-double--bottom">
-    <thead>
-        <tr>
-            <th>{{ 'question' | trans({}, 'common') }}</th>
-            <th>{{ 'answer' | trans({}, 'common') }}</th>
-            <th>
-                <span class="visually-hidden">{{ 'actions' | trans({}, 'common') }}</span>
-            </th>
-        </tr>
-    </thead>
-    <tbody>
-        <tr class="behat-region-has-assets">
-            <td>
-                {{ 'existPage.form.noAssetToAdd.label' | trans(transOptions)  }}
-            </td>
-            <td>
-                {{ ndr.noAssetToAdd ? 'No' : 'Yes' }}
-            </td>
-            <td class="change-answer">
-                <a href="{{ url('ndr_assets_exist', { 'ndrId': ndr.id, 'from': 'summary' }) }}" class="behat-link-edit">{{ 'edit' | trans({}, 'common') }}</a>
-            </td>
-        </tr>
-    </tbody>
-</table>
-
+<dl class="govuk-summary-list columns-2 push-double--bottom">
+  <div class="govuk-summary-list__row">
+    <dt class="govuk-summary-list__key">
+      {{ 'question' | trans({}, 'common') }}
+    </dt>
+    <dd class="govuk-summary-list__key">
```

Comment:
This should be `govuk-summary-list__value`. Again, should be straightforward to search for others like this